atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -83,8 +82,8 @@ def run(self):
                 self.workflow.dockerfile_images.custom_base_image):
             self._base_image_nvr = self.detect_parent_image_nvr(
                 self.workflow.dockerfile_images.base_image,
-                # OSBS2 TBD
-                inspect_data=imageutil.base_image_inspect(),
+                # OSBS2 TBD: decide if we need to inspect a specific arch
```

Comment:
koji parent is checking for labels to get NVR of brew build

release/version/component will be same for all arches